gammapy/maps/wcsnd.py

```diff
@@ -635,9 +637,10 @@ def cutout(self, position, width, mode='trim', copy=True):
         ----------
         position : `~astropy.coordinates.SkyCoord`
             Center position of the cutout region.
-        width : tuple of `~astropy.coordinates.Angle`
-            Angular sizes of the region in (lon, lat). If only one value is passed,
-            a square region is extracted. For more options see also `~astropy.nddata.utils.Cutout2D`.
+        width : tuple or list of `~astropy.coordinates.Angle`
```

Comment:
Remove mention of "list" here, we just support length 2 tuples and scalars (please don't add more options).
Suggest to remove the link to Cutout2D now, as our options are now different and reading their width will just confuse users. It's now an implementation detail that might even change later.